index.html

```diff
@@ -1,218 +1,259 @@
+          </a>
+        </li>
+      </ul>
+    </header>
+    <main>
+      <section class="top">
+        <h1 class="profile">I'm Damilare Glad to see you!</h1> 
+        <p class="dev">
+         I’m a software developer! I can help you build a product , feature or 
+         website Look through some of my work and experience! If you like what 
+         you see and have a project you need coded, don’t hestiate to contact 
+         me.
+        </p>
+        <div><a href="#" class="connect" >LET'S CONNECT</a></div>
+        <ul>
+          <li>
+            <a href="#">
+              <img src="./images/Twitter-logo.png"  alt="twitter">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/Linkedin-logo.png" alt="linkedin">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/M-logo.png" alt="letter m">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/Github-logo.png" alt="github">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/fingers-logo.png" alt="finger icon">
+            </a>
+          </li>
+        </ul>
+      </section>
+    </main>
+    <section class="work-section">
+      <ul class="project">
+        <li class="project-card">
+          <div class="card-img">
+            <img src="./images/Snapshoot-Portfolio.png" class="pix" alt="snap">
+            <img src="./images/first-snapshot-desktop.png" alt="desktop" class="snap">
+          </div>
+          <div class="card-content">
+            <h1 class="project-title">Tonic</h1>
+            <p class="project-description">CANOPY
+              <span><img src="./images/rename.png" alt="period">Back End Dev </span>
+              <span><img src="./images/rename.png" alt="period">2015</span>
+            </p>
+            <p class="project-content">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="lang">
+              <li class="btn">html</li>
+              <li class="btn">css</li>
+              <li class="btn">javascript</li>
+            </ul>
+            <div>
+              <button type="button" class="bon">See Project</button>
+            </div>
+          </div>
+        </li>
+        <li class="project-card">
+          <div class="card-img flex-order">
+            <img src="./images/project-1.png" class="pix" alt="snap">
+            <img src="./images/second-snapshot-desktop.png" alt="desktop" class="snap">
+          </div>
+          <div class="card-content">
+            <h1 class="project-title">Multi-Post Stories</h1>
+            <p class="project-description">CANOPY
+              <span><img src="./images/rename.png" alt="period">Back End Dev </span>
+              <span><img src="./images/rename.png" alt="period">2015</span>
```

Comment:
- [ ] This section need some more margin and padding. Comparing yours with figma...

### your design

![Screenshot from 2021-06-11 20-27-27](https://user-images.githubusercontent.com/22434660/121707122-e2734180-caf3-11eb-91a7-45d9148755f6.png)


### figma design

![Screenshot from 2021-06-11 20-28-03](https://user-images.githubusercontent.com/22434660/121707140-e8692280-caf3-11eb-8cf0-23933a3a7eb5.png)

Did you see the difference :question: :question: :question: